### PR TITLE
Don't hardcode part of the path for customizable CoreCLROverridePath

### PR DESCRIPTION
**Pull request by marek-safar**

### CODE REVIEW

File: external/runtime/runtime.depproj [hunk 1]
```diff
@@ -108,7 +108,7 @@
   <Target Name="GetCoreCLRILFiles" DependsOnTargets="GetCoreLibPackagePath">
     <ItemGroup>
       <CoreCLRILFiles Condition="'$(CoreCLROverridePath)' == ''" Include="$(_CoreLibPackagePath)/**/il/*.*" />
-      <CoreCLRILFiles Condition="'$(CoreCLROverridePath)' != ''" Include="$(CoreCLROverridePath)/IL/*.*" />
+      <CoreCLRILFiles Condition="'$(CoreCLROverridePath)' != ''" Include="$(CoreCLROverridePath)/*.*" />
```
Comment (ViktorHofer) [inline external/runtime/runtime.depproj]: The IL files are expected to be located in a sub directory. This change would pick up all the assemblies.

Comment (marek-safar) [reply external/runtime/runtime.depproj]: No, if you include the full path. We are using this to verify that CoreFX builds with Mono's version of System.Private.CoreLib. This was changed recently and with this change, you cannot specify the full path as it always appends /IL to it.

Comment (ViktorHofer) [reply external/runtime/runtime.depproj]: This target `GetCoreCLRILFiles` expects IL files to be present in an IL directory. This is how we do local coreclr testing + code coverage where we need the IL CoreLib assembly. Do you need that target at all when running with MonoVM?

Please not that I'm going to make more changes to `runtime.depproj` to support MonoVM (+CI) as soon as some blockers in the infra are resolved.

Comment (marek-safar) [reply external/runtime/runtime.depproj]: There are two things. I think if you allow having an overrideable property it should be fully overrideable not just partially. Secondly, I'd like to see automated CI testing for CoreFX interface for System.Private.CoreLib for all runtimes. We are doing that manually by setting CoreCLROverridePath right now but once we progress on CI integration it should be automated as well.

**Comment by maryamariyan** [comment]
@ViktorHofer @marek-safar this PR was inactive since late May.
What is required next to do for this PR?

**Comment by maryamariyan** [comment]
@marek-safar closing this PR due to inactivity. Please feel free to reopen when ready.

File: external/runtime/runtime.depproj (hunk 1 above)
Comment (ViktorHofer) [reply external/runtime/runtime.depproj]: Marek to push this forward I would like to explain this in more detail. `CoreCLROverridePath` points to the location where the native "cross-gend" CLR bits are. Additionally to that we also need the IL version of the CLR to allow code coverage measurement. The latter lives in a subdirectory "IL". Does Mono have both cross-gend and IL versions of CoreLib? We would need both.

Comment (marek-safar) [reply external/runtime/runtime.depproj]: I don't know what "cross-gend" bits are needed for but I was able to build CoreFX with just CoreCLROverridePath set to managed dll which is what we used it for to test API matches

Comment (ViktorHofer) [reply external/runtime/runtime.depproj]: Does that mean you only have one set of libraries - managed ones?

Comment (marek-safar) [reply external/runtime/runtime.depproj]: Yes

**Pull request closed without merging**